drivers/hmis_csv_importer/app/jobs/hmis_csv_importer/cleanup/expire_base_job.rb

```diff
@@ -48,11 +48,17 @@ def perform(model_name: nil, retain_item_count: 5, retain_after_date: DateTime.c
       return unless next_model
 
       # enqueue the job for the next model in series
-      self.class.perform_later(
+      options = {
         model_name: next_model.name,
         retain_item_count: retain_item_count,
         retain_after_date: retain_after_date,
-      )
+        delay: delay,
+      }
+      if delay
+        self.class.perform_later(**options)
+      else
+        self.class.perform_now(**options)
```

Comment:
Assume this is for development? Not sure that supporting the delay option is worth it just for dev. Also performing the job inline means you'd get a stack of these jobs (like ~50) and could get weird interactions cause they are all pending cleanup. Suggest just firing up a local worker instead for development

Reply:
oh I see, this is because we aren't running in dj anymore

Reply:
Yeah, this isn't necessary, but seemed like a reasonable way to let us play around with DJ vs cron without having to push a code change.